content/gettingstarted/index.adoc

```diff
@@ -22,123 +22,125 @@ Kiali will support the latest Istio version (1.1 at the moment) and the developm
 
-You can also use the `oc` command to determine the base-url:
+=== Kiali Operator
 
-[source,bash]
-----
-oc get route -n istio-system -l app=kiali
-----
+Now install the latest release of Kiali using the Kiali Operator.
 
-== Getting started on Kubernetes
+==== What is the Kiali Operator?
 
-To deploy Kiali to your Kubernetes cluster you can do the following.
+First, a quick introduction of what the Kiali Operator actually is.
 
-=== Preparation
+The Kiali Operator is an implementation of a link:https://coreos.com/operators/[Kubernetes Operator]. It provides a method of packaging, deploying, and managing Kiali.
 
-Preparation is only needed if you are installing in Minikube. When installing in Minikube, ensure that you enable the Ingress add-on by executing the following command:
+The Kiali Operator runs inside the `kiali-operator` namespace and listens for the creation of a Kiali CR (custom resource) object. When it sees one created, it will install Kiali using the settings it finds in the CR object. To see all the different settings you can configure in a Kiali CR, see link:https://github.com/kiali/kiali/blob/master/operator/deploy/kiali/kiali_cr.yaml[this fully documented example Kiali CR yaml file]. A minimal Kiali CR that falls back to all defaults looks like this:
 
-[source,bash]
+[source,yaml]
 ----
-minikube addons enable ingress
+apiVersion: kiali.io/v1alpha1
+kind: Kiali
+metadata:
+  name: kiali
 ----
 
-You can now install Istio if needed. See the link:https://istio.io/docs/setup/[Istio Setup docs] for details.
+==== Installing the Kiali Operator
 
-=== Install Kiali
+In the instructions below, you will be shown how to install the Kiali Operator using the link:https://github.com/kiali/kiali/blob/master/operator/deploy/deploy-kiali-operator.sh[Kiali Operator install script]. That install script has the option of installing a Kiali CR for you, thus making it easy to get started with Kiali by running a single command (this is, in fact, its default behavior). If, however, you wish to configure Kiali with specific settings of your choosing, you can tell the script to skip the installation of a Kiali CR and just simply install the Operator. In this case, you would be responsible for writing the Kiali CR yaml file and creating it in your cluster.
 
-Now install the latest release of Kiali.
+icon:lightbulb[size=1x]{nbsp} The Kiali Operator install script can be configured by setting environment variables when running it. For example, if you do not want it to install a Kiali CR (and thus avoid having the Kiali Operator install Kiali itself), set the OPERATOR_INSTALL_KIALI environment variable to "false" and then run the script. Read the comments at the top of the link:https://github.com/kiali/kiali/blob/master/operator/deploy/deploy-kiali-operator.sh[Kiali Operator install script] for details on what custom configuration settings are available to you.
```

Comment:
this part is not very clear. if someone doesn't want Kiali, then they still install the Kiali operator and disable something? what's the purpose?

Reply:
Yes, this is going to be confusing for anyone who isn't familiar with operators. I can't figure out a good way to word this. But I'll try again.

This is actually the reason why I added the "What is the Kiali Operator?" section - to try to introduce operators which would then hopefully make it more understandable.

What this is trying to say is - if you don't have a specific Kiali configuration that you want (maybe you just want to kick the tires with a demo and just quickly get something up and running? Maybe you want to test Kiali running with all of its settings set to their defaults?), then this script can install the operator AND kiali together for you. If, on the other hand, you have a specific configuration you want Kiali to have, then don't tell this script to install Kiali for you. Just install the operator and then you create the Kiali CR with the configuration you want and deploy that Kiali CR manually yourself.

In other words, this script will provide you with a base Kiali installation *only as a convienence* (i.e. so you don't have to create the Kiali CR yaml and deploy it yourself). But, if the base "all defaults" Kiali installation is configured in a way you don't want, then you can just tell this script to only install the operator and you will tell the operator later how you want kiali installed.

For example, you may want to install Kiali with a specific Jaeger URL, Grafana URL, web root, and you want to disable metrics and you want to enable view_only mode. Well, you want a very specifically configured Kiali - not this "base" Kiali that uses all defaults. In this case, tell this install script to only install the operator and skip the Kiali installation. Then you can write your own Kial CR yaml file (with all those settings you want to customize) and then you can manually "oc create -f my-kiali-cr.yaml -n kiali-operator" which will then tell the operator to install kiali with that special configuration.

Now I just have to figure out how to word all this properly :)

Reply:
I added a little bit extra here.